pkg/controller/performanceprofile/components/consts.go

```diff
@@ -26,5 +28,7 @@ const (
 
 const (
 	// FeatureGateLatencySensetiveName defines the latency sensetive feature gate name
-	FeatureGateLatencySensetiveName = "latency-sensetive"
-)
\ No newline at end of file
+	// TOOD: uncomment once https://bugzilla.redhat.com/show_bug.cgi?id=1788061 fixed
+	// FeatureGateLatencySensetiveName = "latency-sensetive"
+	FeatureGateLatencySensetiveName = "cluster"
```

Comment:
it does not feature set, it the name of the feature gate resource.
The initial idea was to create additional feature gate resource that should enable topology manager(that will be managed by us), but because of the bug, the kubelet controller does not render correct machine config(it uses only default `cluster` feature gate resource)

Reply:
fixed typo

Reply:
> but because of the bug, the kubelet controller does not render correct machine config(it uses only default cluster feature gate resource)

oh, so does this mean we can't enable topology manager until that BZ is resolved?

Reply:
nope, it just means that we should update the default feature gate resource `cluster` instead of creating a new one(that can create some conflicts with user configuration, so I wanted to avoid it), also it should be updated before the kubelet config creation, see - https://bugzilla.redhat.com/show_bug.cgi?id=1788061#c3